[CI] LogsdbIndexingRollingUpgradeIT testIndexing {upgradedNodes=2} failing

**Build Scans:**
- [elasticsearch-intake #20048 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/tmv3x5wtmxqd2)
- [elasticsearch-pull-request #62598 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/skkbcrnhct2hg)
- [elasticsearch-pull-request #62595 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/2sc7vt6vrj6ic)
- [elasticsearch-pull-request #62597 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/u7kpiq3ddzhu6)
- [elasticsearch-pull-request #62592 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/jncp3qirpdhuw)
- [elasticsearch-intake #20046 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/3je2bqs3i7zkq)
- [elasticsearch-pull-request #62590 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/wv7y2pgb4dlfw)
- [elasticsearch-pull-request #62589 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/2ztic45q6b4ww)
- [elasticsearch-intake #20044 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/cocx3p5wvv7mi)
- [elasticsearch-pull-request #62581 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/kczublcxyt56e)

**Reproduction Line:**
```
./gradlew ":qa:rolling-upgrade:v9.0.0#bwcTest" -Dtests.class="org.elasticsearch.upgrades.LogsdbIndexingRollingUpgradeIT" -Dtests.method="testIndexing {upgradedNodes=1}" -Dtests.seed=BC808F086923CB28 -Dtests.bwc=true -Dtests.locale=es-PA -Dtests.timezone=Pacific/Fiji -Druntime.java=23
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.upgrades.LogsdbIndexingRollingUpgradeIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testIndexing%20%7BupgradedNodes%3D2%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: timed out waiting for green state for index [logs-bwc-test] cluster state [{
  "cluster_name" : "test-cluster",
  "cluster_uuid" : "bDRMLq-FTiaBRwWDcOp73Q",
  "version" : 92,
  "state_uuid" : "wJZwpQztSJSWdertfdwIqA",
  "master_node" : "V20gptq2RlyMtqOAmf0Qgw",
  "blocks" : { },
  "nodes" : {
    "V20gptq2RlyMtqOAmf0Qgw" : {
      "name" : "test-cluster-1",
      "ephemeral_id" : "45sNebf6Rr6Vw-t2H7lMQw",
      "transport_address" : "127.0.0.1:39141",
      "external_id" : "test-cluster-1",
      "attributes" : {
        "testattr" : "test",
        "transform.config_version" : "10.0.0",
        "xpack.installed" : "true",
        "ml.allocated_processors" : "32",
        "ml.max_jvm_size" : "536870912",
        "ml.config_version" : "12.0.0",
        "ml.machine_memory" : "101237063680",
        "ml.allocated_processors_double" : "32.0"
      },
      "roles" : [
        "data",
        "data_cold",
        "data_content",
        "data_frozen",
        "data
[truncated]
```

**Issue Reasons:**
- [main] 7 consecutive failures in step 9.0.0_bwc-snapshots
- [main] 16 failures in test testIndexing {upgradedNodes=2} (1.7% fail rate in 947 executions)
- [main] 13 failures in step 9.0.0_bwc-snapshots (4.5% fail rate in 286 executions)
- [main] 2 failures in step 9.1.0_bwc-snapshots (0.7% fail rate in 288 executions)
- [main] 3 failures in pipeline elasticsearch-intake (4.5% fail rate in 67 executions)
- [main] 11 failures in pipeline elasticsearch-pull-request (4.7% fail rate in 236 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -339,15 +339,6 @@ tests:
 - class: org.elasticsearch.common.metrics.ExponentiallyWeightedMovingRateTests
   method: testEwmr_threadSafe
   issue: https://github.com/elastic/elasticsearch/issues/124692
-- class: org.elasticsearch.upgrades.LogsdbIndexingRollingUpgradeIT
-  method: testIndexing {upgradedNodes=2}
-  issue: https://github.com/elastic/elasticsearch/issues/124830
-- class: org.elasticsearch.upgrades.LogsdbIndexingRollingUpgradeIT
-  method: testIndexing {upgradedNodes=3}
-  issue: https://github.com/elastic/elasticsearch/issues/124831
-- class: org.elasticsearch.upgrades.LogsdbIndexingRollingUpgradeIT
-  method: testIndexing {upgradedNodes=1}
-  issue: https://github.com/elastic/elasticsearch/issues/124833
 - class: org.elasticsearch.xpack.ilm.DataStreamAndIndexLifecycleMixingTests
   method: testUpdateIndexTemplateToDataStreamLifecyclePreference
   issue: https://github.com/elastic/elasticsearch/issues/124837

